WorkloadManagementSystem/Service/WMSUtilities.py

```diff
@@ -78,12 +79,11 @@ def getARCPilotOutput( proxy, pilotRef ):
   myce = pilotRef.split(":")[1].strip("/")
   gridEnv = getGridEnv()
   mySite = getCESiteMapping()['Value'][myce]
-  WorkDB = gConfig.getValue(os.path.join('Resources/Sites/LCG', mySite, 'CEs', myce, 'JobListFile'))
-  myWorkDB = os.path.join("/opt/dirac/runit/WorkloadManagement/SiteDirector-RAL2", WorkDB)
+  workDB = gConfig.getValue(cfgPath(['Resources/Sites/LCG', mySite, 'CEs', myce, 'JobListFile']))
```

Comment:
This line must be just:
workDB = gConfig.getValue( cfgPath( 'Resources/Sites/LCG', mySite, 'CEs', myce, 'JobListFile' ) )